sources/scraping/scrapy/pipeline_item.py

```diff
@@ -0,0 +1,46 @@
+from abc import ABC
```

Comment:
@rudolfix I opted to use a closure function to return scrapy item pipeline and give access to queue instance to internal class